sArena.toc

```diff
@@ -1,8 +1,8 @@
-## Interface: 90005
+## Interface: 100000
 ## Title: sArena
 ## Notes: Enhanced arena frames
 ## Author: Stako
-## Version: 3.1.8
+## Version: 3.1.9
```

Comment:
Make this 3.2 to signify a more significant change (backward incompatible with old version)